docs/source/developer/interface/command-line-parameters.md

```diff
@@ -0,0 +1,35 @@
+# Command Line Parameters
+
+You can launch an interface with these parameters.
+
```

Comment:
There are further command line parameters per: https://wiki.fluffy.ws/wiki/Interface_CLI_Parameters

Reply:
I realize now I never inputted the interface CLI parameters properly, I corrected this now.